apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-item-presenter.tsx

```diff
@@ -0,0 +1,27 @@
+import { IMultilingualTextItem } from '@coscrad/api-interfaces';
+import LanguageIcon from '@mui/icons-material/Language';
```

Comment:
Update icon import to `@mui/icons-material`

Reply:
You beat me to it!